VideoLocker/res/values/colors.xml

```diff
@@ -31,13 +31,14 @@
     <color name="transparent_black_80">#CC000000</color>
     <color name="transparent_black_90">#e5000000</color>
     <color name="transparent_black_50">#80000000</color>
-    <color name="transparent_black_10">#19000000</color>
+    <color name="transparent_black_26">#42000000</color>
+    <color name="transparent_black_13">#21000000</color>
     <color name="transparent_white_90">#E6FFFFFF</color>
     <color name="transparent_white_85">#D9FFFFFF</color>
     <color name="transparent_white_30">#4DFFFFFF</color>
     <color name="transparent_white_26">#42FFFFFF</color>
     <color name="transparent_white_20">#33FFFFFF</color>
-    <color name="transparent_white_10">#19FFFFFF</color>
+    <color name="transparent_white_13">#21ffffff</color>

```

Comment:
What's up with these names? I know you weren't the person who introduced this naming scheme, but it makes it difficult. I guess I have no actionable complain here.